gemma-web/src/main/java/ubic/gemma/web/services/rest/providers/RequestExceptionEventListener.java

```diff
@@ -11,7 +12,7 @@
  * @author poirigui
  */
 @Provider
-public class ApplicationEventListener implements org.glassfish.jersey.server.monitoring.ApplicationEventListener {
+public class RequestExceptionEventListener implements ApplicationEventListener {
```

Comment:
This needs to be renamed because we're introducing another `ApplicationEventListener`.